From 99954b4ebdbee382006c6afc804c5196a1527e8c Mon Sep 17 00:00:00 2001
From: Jin Wei <wei.a.jin@intel.com>
Date: Thu, 18 Oct 2012 17:04:21 +0800
Subject: fix valgrind ioctl warning.

BZ: 62797
[CLEAN]

Initialize local variable to avoid syscall ioctl warning.

Change-Id: Ic1d0daeaf03ca3f60dfbe1aa6601cc47bc75447e
Orig-Change-Id: I6893f17ab8b036c8723707fba2b2bdcf49cae075
Category: aosp improvement
Domain: AOSP-General
Origin: internal
Upstream-Candidate: yes
Signed-off-by: Jin Wei <wei.a.jin@intel.com>
---
 libcutils/ashmem-dev.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libcutils/ashmem-dev.c b/libcutils/ashmem-dev.c
index 8b71f87..3089a94 100644
--- a/libcutils/ashmem-dev.c
+++ b/libcutils/ashmem-dev.c
@@ -48,7 +48,7 @@ int ashmem_create_region(const char *name, size_t size)
 		return fd;
 
 	if (name) {
-		char buf[ASHMEM_NAME_LEN];
+		char buf[ASHMEM_NAME_LEN] = {0};
 
 		strlcpy(buf, name, sizeof(buf));
 		ret = ioctl(fd, ASHMEM_SET_NAME, buf);
-- 
1.7.4.1

